src/utils/preprocessing.py

```diff
@@ -107,11 +110,14 @@ def get_depthmaps(fpaths):
     return depthmaps
 
 
-def standing_laying_data_preprocessing(source_path):
+def standing_laying_data_preprocessing(source_path, scan_type):
     img = tf.io.read_file(str(source_path))
     img = tf.image.decode_jpeg(img, channels=3)
     img = tf.cast(img, tf.float32) * (1. / 256)
-    img = tf.image.rot90(img, k=3)
+    if scan_type is standing_scan_type:
+        img = tf.image.rot90(img, k=3)
+    elif scan_type is standing_scan_type:
+        img = tf.image.rot90(img, k=1)
```

Comment:
You could add an else branch and define what should happen if scan_type is in neither of the lists. 

I think this would make the code safer if you deal with this edge case